src/components/Map/index.tsx

```diff
@@ -282,7 +282,6 @@ const Map = () => {
           type="full-screen"
           onClick={() => handleFullScreenChange(cityDustInfoMarkers)}
         />
-        <ControlButton type="go-back" onClick={handleClickGoBack} />
         {zoomLevel === MAX_ZOOM_LEVEL && !sidoDustInfoList && <Spinner />}
       </VStack>
       {!cityDustInfoList ? <Spinner zIndex={10} /> : ''}
```

Comment:
이 부분은 빈 문자열 대신 `null`을 반환하는게 맞지 않을까? 하는 생각.

Reply:
혹시 `null`을 반환하면 null 이라고 출력되진 않나요?

Reply:
```suggestion
      {!cityDustInfoList && <Spinner zIndex={10} />}
```
빈 문자열보다 요렇게 아무것도 반환하지 않도록 수정하는게 좋을 것 같아요~!